test/Blockfrost/GenerateFixtures/NativeScript.purs

```diff
@@ -0,0 +1,116 @@
+module Test.Ctl.Blockfrost.GenerateFixtures.NativeScript (main) where
+
+import Contract.Prelude
+
+import Contract.Address (ownPaymentPubKeyHash, ownStakePubKeyHash)
+import Contract.Config
+  ( ContractParams
+  , PrivatePaymentKeySource(PrivatePaymentKeyFile)
+  , WalletSpec(UseKeys)
+  , testnetConfig
+  )
+import Contract.Hashing (scriptRefHash) as Hashing
+import Contract.Monad
+  ( Contract
+  , launchAff_
+  , liftedM
+  , runContract
+  )
+import Contract.ScriptLookups (ScriptLookups) as Lookups
+import Contract.Scripts (NativeScript, ScriptHash)
+import Contract.Transaction
+  ( ScriptRef(NativeScriptRef)
+  , awaitTxConfirmed
+  , submitTxFromConstraints
+  )
+import Contract.TxConstraints (TxConstraints) as Constraints
+import Contract.Value (lovelaceValueOf) as Value
+import Ctl.Examples.Helpers (mustPayToPubKeyStakeAddressWithScriptRef)
+import Ctl.Internal.Contract.QueryBackend (BlockfrostBackend)
+import Ctl.Internal.Service.Blockfrost
+  ( BlockfrostEndpoint(NativeScriptByHash)
+  , BlockfrostRawResponse
+  , runBlockfrostServiceTestM
+  )
+import Ctl.Internal.Service.Blockfrost (getScriptByHash) as Blockfrost
+import Data.Array (mapWithIndex)
+import Data.BigInt (fromInt) as BigInt
+import Test.Ctl.Blockfrost.GenerateFixtures.Helpers
+  ( blockfrostBackend
+  , getSkeyFilepathFromEnv
+  , storeBlockfrostFixture
+  )
+import Test.QuickCheck.Arbitrary (arbitrary)
+import Test.QuickCheck.Gen (randomSample')
+import Test.Spec.Assertions (shouldEqual)
+
+main :: Effect Unit
+main =
+  contractParams >>=
+    (launchAff_ <<< flip runContract (generateFixtures 10))
+  where
+  contractParams :: Effect ContractParams
+  contractParams = do
+    -- blockfrostApiKey <- lookupEnv' "BLOCKFROST_API_KEY"
+    skeyFilepath <- getSkeyFilepathFromEnv
+    pure $ testnetConfig
+      -- TODO: Configure Contract with Blockfrost as the default backend.
+      -- { backendParams =
+      --    mkBlockfrostBackendParams
+      --      { blockfrostConfig: blockfrostPublicPreviewServerConfig
+      --      , blockfrostApiKey: Just blockfrostApiKey
+      --      }
```

Comment:
Leftover comment and TODO

Reply:
It is not possible to use Blockfrost as the default backend until we set the ledger "constants" for Blockfrost backend:
https://github.com/Plutonomicon/cardano-transaction-lib/blob/b7456bbcc20fa94efd776ff02848116d050ed1fe/src/Internal/Contract/Monad.purs#L237-L242